src/features/unified-share-modal/UnifiedShareModal.scss

```diff
@@ -3,78 +3,64 @@
 @import '../../styles/mixins/overlay';
 @import '../../components/pill-selector-dropdown/mixins';
 
-.is-responsive-web {
```

Comment:
the changes in this file are relatively the same but with two main differences:
- `is-responsive-web` outer scope is removed
- `collaborator-list` styles are moved to `CollaboratorList.scss`

Reply:
nvm, I also included other fixes:
- moved the breakpoints to the end of the file so that if any rules have the same specificity, the breakpoint ones will be applied over the regular screen size
- added a `unified-share-modal` scope to several blocks so that they have at least the same specificity level as the regular styles in this stylesheet
- reorganized nesting of several blocks to mirror the specificity levels as the regular styles

the intention of these changes is that is should make it easer to maintain them in the future since the nesting and organization of the regular and breakpoint styles are now similar